docs/user/reference/general/initializing.md

```diff
@@ -67,6 +67,7 @@ Below are listed the configuration options available on most SDKs.
 - `httpUploader`: A custom HTTP uploader instance, that will overwrite Glean's provided uploader. Useful
 for users that wish to use specific uploader implementations. See [Custom Uploaders](#custom-uploaders)
 for more information on how and when the use this feature.
+- `logLevel`: The log level for the `log` rust crate. The level filter options in order from least to most verbose are: `Off`, `Error`, `Warn`, `Info`, `Debug`, `Trace`. The default option if nothing is provided is `Trace`. See the [`log` crate docs](https://docs.rs/log/latest/log/) for more information.
```

Comment:
Trace does seem a bit _extra_ verbose. I don't actually think we log anything at that level anyway. I think `Debug` is plenty verbose for our default.